packages/plans-grid-next/src/components/comparison-grid/index.tsx

```diff
@@ -297,6 +279,24 @@ const PlanSelector = styled.header`
 			outline: thin dotted;
 		}
 	}
+
+	.plan-comparison-grid__title--is-select-trigger {
+		display: flex;
+	}
+
+	.plan-comparison-grid__title-icon {
+		position: relative;
+		top: -2px;
+		left: -2px;
+		width: 1px;
+		height: 1px;
+		overflow: visible;
+
+		svg {
+			fill: var( --color-link );
+			transform: rotate( 90deg );
```

Comment:
I ended up using `@wordpress/icons/chevronRightSmall` and then rotating it 90 degrees, because `chevronUpSmall`/`chevronDownSmall` don’t exist.

Reply:
Hmm clicking on the icon no longer works for me, only the plan name reacts. it's likely the re-styling done here that's not wrapping the icon properly.

Reply:
it was an order thing with the SVG sitting above the select